Userland/Shell/Job.h

```diff
@@ -31,7 +27,7 @@ class Job : public RefCounted<Job> {
 
     ~Job()
     {
-#ifdef JOB_TIME_INFO
+#if SHELL_JOB_DEBUG
```

Comment:
n i t:
```suggestion
if constexpr (SHELL_JOB_DEBUG) {
```

Reply:
(and replace the `#endif` below with the good old `}`)

Reply:
le fixed, and moved the macro in AK/Debug.in.h to its proper sorted location at the same time